test/const_node_inspector_test.rb

```diff
@@ -5,7 +5,7 @@
 require "parser_test_helper"
 
 module Packwerk
-  class ConstNodeInspectorTest < ActiveSupport::TestCase
+  class ConstNodeInspectorTest < Minitest::Test
```

Comment:
why this change?

Reply:
The tests in these two classes started writing formatted output, probably as a result of the Dummy app being initialized. This initially created a log file in the dummy skeleton until I changed it to `stdout`. To silence them completely, I figured I'd just make them consistent with the rest of the tests.
Alternatively, maybe we could configure the Dummy app with some sort of `NullLogger` I suppose?

Reply:
That seems like a good solution 🤔  TBH I don't remember why these tests are using `ActiveSupport::TestCase` and others do not. But it seems like it would be confusing for someone to start using `ActiveSupport::TestCase` and thereby inadvertently generating logs.

Reply:
Done in 805a77d14179e742ada7ff7a51ad6f19171e7329 👍